### PR TITLE
Update guide to show how to publish messages to Slack

### PR DESCRIPTION
**Pull request by lucamolteni**
No Jira

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

* <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

* <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: DEMO.md [hunk 1]
```diff
@@ -201,6 +201,46 @@ Like for the Bridge instance, it is needed to wait until the Processor has been
...
+      "webhookUrl": "webhookURL"
+    }
+  }
+}
+```
+
+Here's the example payload of the `Webhook` action instead. The channel is not needed but a specific transformation template that will transform the CloudEvents to Slack's required format is needed. Messages that don't comply to this format won't be written on Slack when using the `Webhook` action.
+
+```json
+{"text":"Hello, World!"}
```
Comment (manstis) [inline DEMO.md:226]: What's this for?

**Review by robpblake — APPROVED** [review]
LGTM!

File: DEMO.md [hunk 2]
```diff
@@ -201,6 +201,46 @@ Like for the Bridge instance, it is needed to wait until the Processor has been
...
+    "type": "Slack",
+    "parameters": {
+      "channel": "channel",
+      "webhookUrl": "webhookURL"
+    }
+  }
+}
+```
+
+Here's the example payload of the `Webhook` action instead. The channel is not needed but a specific transformation template that will transform the CloudEvents to Slack's required format is needed. Messages that don't comply to this format won't be written on Slack when using the `Webhook` action.
```
Comment (manstis) [inline DEMO.md:223]: The `CloudEvents` what? Payload? or simply "...will transform the `CloudEvent` to Slack's..."

File: DEMO.md (hunk 1 above)
Comment (lucamolteni) [reply DEMO.md:226]: To publish messages to Slack!

File: DEMO.md (hunk 2 above)
Comment (lucamolteni) [reply DEMO.md:223]: Unfortunately `Cloudevents` is both plural and a noun :( see https://cloudevents.io

I'll write Cloudevents data instead

**Review by manstis — APPROVED** [review]

**Pull request merged**